templates/dataset.html

```diff
@@ -13,6 +13,15 @@ <h3>Tags: {{ dataset.tags|join(", ") }}</h3>
 
     <p>{{ dataset.description }}</p>
 
+    {% if let Some(comment) = dataset.comment %} <p>Comment: {{ comment }}</p> {% endif %}
+
+    {% if let Some(last_checked) = dataset.last_checked %} <p>Last checked on source metadata portal: {{ last_checked }}</p> {% endif %}
+
+    {% if let Some(region) = dataset.region %} <p>Region: {{ region }}</p> {% endif %}
+
+    <p>      Contact: {% for contact in dataset.contact_names %}  {{ contact }}, {% endfor %}  </p>
```

Comment:
Whitespace is mostly meaningless in HTML, so this should not be rendered differently if you leave out the space after `<p>` and before </p>`.

Concerning the for loop, I think this actually needs the `{{ ...|join(", ") }}` filter?

Reply:
Thanks, I'll take that suggestion.